src/js/components/DialogsComponent.jsx

```diff
@@ -40,7 +41,9 @@ var DialogsComponent = React.createClass({
       DialogEvents.CONFIRM_SHOW,
       this.onDialogConfirmShow
     );
-    DialogStore.removeListener(DialogEvents.CONFIRM_DISMISS, this.onDialogClose);
+    DialogStore.removeListener(
+      DialogEvents.CONFIRM_DISMISS, this.onDialogClose
```

Comment:
Yes, I also prefer one of those two from you @pierlo-upitup , but unsure which one..